src/Core/Protocol.php

```diff
@@ -205,7 +206,8 @@ public static function unfollow(array $contact, array $user): ?bool
 			$slap = OStatus::salmon($item, $user);
 
 			if (empty($contact['notify'])) {
-				throw new \InvalidArgumentException('Missing expected "notify" key in OStatus/DFRN contact');
+				Logger::notice('OStatus/DFRN Contact is missing notify, we quit here', ['id' => $contact['id']]);
+				return true;
```

Comment:
Please return `null` instead, as no remote action was performed. A `null` return value will not defer the Unfollow task.